apps/version.c

```diff
@@ -133,6 +137,25 @@ int version_main(int argc, char **argv)
         printf("%s\n", OpenSSL_version(OPENSSL_DIR));
     if (engdir)
         printf("%s\n", OpenSSL_version(OPENSSL_ENGINES_DIR));
+    if (seed) {
+        printf("Seeding source:");
+#ifdef OPENSSL_RAND_SEED_GETRANDOM
+        printf(" getrandom syscall");
+#endif
+#ifdef OPENSSL_RAND_SEED_DEVRANDOM
+        printf(" /dev random");
```

Comment:
It should never be /dev/random on Linux. Maybe use /dev/*random in the printf or actually use the DEVRANDOM constant here.